docs/source/overview_software_architecture.rst

```diff
@@ -88,7 +87,7 @@ Spec
    * :py:class:`pynwb.spec.NWBDatasetSpec` - specification for dataset (like and n-dimensional array). Specifies data type, dimensions, etc.
    * :py:class:`pynwb.spec.NWBLinkSpec` - specification for link (like a POSIX soft link)
    * :py:class:`pynwb.form.spec.spec.RefSpec` - specification for references (References are like links, but stored as data)
-   * :py:class:`pynwb.spec.NWBDtypeSpec` - specification for compound data types. Used to build complex data type specification, e.g., to define tables (used only in ``DatasetSpec``)
+   * :py:class:`pynwb.spec.NWBDtypeSpec` - specification for compound data types. Used to build complex data type specification, e.g., to define tables (used only in :py:class:`DatasetSpec <pynwb.spec.NWBDatasetSpec>`)
```

Comment:
Can you change this to "... (used only in :py:class:`DatasetSpec <pynwb.from.spec.spec.DatasetSpec>` and correspondingly :py:class:`DatasetSpec <pynwb.spec.NWBDatasetSpec>`)

Reply:
That is right, We should not mix the two of them here.
Also note, that you can use the tilde operator for shortcuts like this:

```rst
(used only in :py:class:`~pynwb.form.spec.spec.DatasetSpec` and
     correspondingly :py:class:`~pynwb.spec.NWBDatasetSpec`)
```
which gives

> (used only in **DatasetSpec** and correspondingly **NWBDatasetSpec**)

While I was at it, I also added the tilde to all of the list items above for overall better readability as the origin of those classes is not important here for the description. And I fixed the mixed line-wrapping here.

Btw.
Is there any `textwidth` setting in your coding guidelines? - In pynwb python files and also rst files it seems to be quite random (looking ugly)